packages/sheets/src/commands/commands/add-worksheet-range-theme.command.ts

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
This file header repeats. There are four occurrences of this. Could you please fix them all?